lib/matplotlib/offsetbox.py

```diff
@@ -498,6 +498,8 @@ class PaddedBox(OffsetBox):
     The `.PaddedBox` contains a `.FancyBboxPatch` that is used to visualize
     it when rendering.
     """
+
+    @_api.make_keyword_only("3.6", name="draw_frame")
```

Comment:
Do you want to start at "pad" here? In AnchoredText/AnchoredBox you chose "pad".

Reply:
I'd leave this as proposed. If in doubt I try to err on the more permissive side, so that we don't overly constrain the API and force people to change still reasonable calls. For `PaddedBox` `pad` is quite canonical and I think we should still allow `PaddedBox(child, 0.5)`. AnchoredText/AnchoredBox have other parameters before pad and pad as central a concept as it is with `PaddedBox`.